tests/test_typing.py::test_mypy fails

```
python3.9 -B setup.py test
running test
/usr/lib/python3.9/site-packages/setuptools/dist.py:634: UserWarning: Usage of dash-separated 'max-line-length' will not be supported in future versions. Please use the underscore name 'max_line_length' instead
  warnings.warn(
copying build/lib.linux-x86_64-3.9/cairo/_cairo.cpython-39-x86_64-linux-gnu.so -> cairo
building 'tests.cmod' extension
creating build/pycairo_tests
creating build/pycairo_tests/temp.linux-x86_64-3.9
creating build/pycairo_tests/temp.linux-x86_64-3.9/tests
creating build/pycairo_tests/temp.linux-x86_64-3.9/tests/cmodule
x86_64-pc-linux-gnu-cc -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -march=native -O2 -pipe -march=native -O2 -pipe -march=native -O2 -pipe -march=native -O2 -pipe -fPIC -DPY_SSIZE_T_CLEAN -Itests/cmodule -Icairo -I/usr/x86_64-pc-linux-gnu/include/cairo -I/usr/x86_64-pc-linux-gnu/include/glib-2.0 -I/usr/x86_64-pc-linux-gnu/lib/glib-2.0/include -I/usr/x86_64-pc-linux-gnu/include/pixman-1 -I/usr/x86_64-pc-linux-gnu/include/freetype2 -I/usr/x86_64-pc-linux-gnu/include/libpng16 -I/usr/x86_64-pc-linux-gnu/include/harfbuzz -I/usr/include/python3.9 -c tests/cmodule/cmodule.c -o build/pycairo_tests/temp.linux-x86_64-3.9/tests/cmodule/cmodule.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
x86_64-pc-linux-gnu-cc -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -march=native -O2 -pipe -march=native -O2 -pipe -march=native -O2 -pipe -march=native -O2 -pipe -fPIC -DPY_SSIZE_T_CLEAN -Itests/cmodule -Icairo -I/usr/x86_64-pc-linux-gnu/include/cairo -I/usr/x86_64-pc-linux-gnu/include/glib-2.0 -I/usr/x86_64-pc-linux-gnu/lib/glib-2.0/include -I/usr/x86_64-pc-linux-gnu/include/pixman-1 -I/usr/x86_64-pc-linux-gnu/include/freetype2 -I/usr/x86_64-pc-linux-gnu/include/libpng16 -I/usr/x86_64-pc-linux-gnu/include/harfbuzz -I/usr/include/python3.9 -c tests/cmodule/cmodulelib.c -o build/pycairo_tests/temp.linux-x86_64-3.9/tests/cmodule/cmodulelib.o -Wall -Warray-bounds -Wcast-align -Wconversion -Wextra -Wformat=2 -Wformat-nonliteral -Wformat-security -Wimplicit-function-declaration -Winit-self -Winline -Wmissing-format-attribute -Wmissing-noreturn -Wnested-externs -Wold-style-definition -Wpacked -Wpointer-arith -Wreturn-type -Wshadow -Wsign-compare -Wstrict-aliasing -Wundef -Wunused-but-set-variable -Wswitch-default -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-command-line-argument -fno-strict-aliasing -fvisibility=hidden -std=c99
creating build/pycairo_tests/lib.linux-x86_64-3.9
creating build/pycairo_tests/lib.linux-x86_64-3.9/tests
x86_64-pc-linux-gnu-cc -pthread -shared -Wl,-O1 -Wl,--as-needed -Wl,-O1 -Wl,--as-needed -Wl,-O1 -Wl,--as-needed -march=native -O2 -pipe -march=native -O2 -pipe build/pycairo_tests/temp.linux-x86_64-3.9/tests/cmodule/cmodule.o build/pycairo_tests/temp.linux-x86_64-3.9/tests/cmodule/cmodulelib.o -L/usr/x86_64-pc-linux-gnu/lib -lcairo -o build/pycairo_tests/lib.linux-x86_64-3.9/tests/cmod.cpython-39-x86_64-linux-gnu.so
copying build/pycairo_tests/lib.linux-x86_64-3.9/tests/cmod.cpython-39-x86_64-linux-gnu.so -> tests
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.9.2, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
benchmark: 3.2.3 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /var/tmp/paludis/build/dev-python-pycairo-1.20.0/work/PYTHON_ABIS/3.9/pycairo-1.20.0
plugins: cov-2.7.1, forked-1.0.1, flake8-1.0.0, xdist-1.26.0, expect-1.1.0, requests-mock-1.7.0, localserver-0.4.1, flaky-3.7.0, benchmark-3.2.3, isort-0.2.0, pyfakefs-4.4.0, hypothesis-5.41.2
collected 276 items / 1 skipped / 275 selected                                                                                                                             

tests/test_api.py .................                                                                                                                                  [  6%]
tests/test_cmod.py .                                                                                                                                                 [  6%]
tests/test_context.py ...................................................................                                                                            [ 30%]
tests/test_device.py ...........                                                                                                                                     [ 34%]
tests/test_enums.py .......                                                                                                                                          [ 37%]
tests/test_error.py ....                                                                                                                                             [ 38%]
tests/test_font.py ..............................                                                                                                                    [ 49%]
tests/test_glyph.py ..                                                                                                                                               [ 50%]
tests/test_hypothesis.py ...........                                                                                                                                 [ 54%]
tests/test_matrix.py ...........                                                                                                                                     [ 58%]
tests/test_path.py ....                                                                                                                                              [ 59%]
tests/test_pattern.py ..............................                                                                                                                 [ 70%]
tests/test_rectangle.py ...                                                                                                                                          [ 71%]
tests/test_region.py ...........                                                                                                                                     [ 75%]
tests/test_surface.py ............................................................                                                                                   [ 97%]
tests/test_surface_numpy.py ..                                                                                                                                       [ 98%]
tests/test_textcluster.py .                                                                                                                                          [ 98%]
tests/test_textextents.py ..                                                                                                                                         [ 99%]
tests/test_typing.py F.                                                                                                                                              [100%]

================================================================================= FAILURES =================================================================================
________________________________________________________________________________ test_mypy _________________________________________________________________________________

    def test_mypy():
        out, err, status = mypy.run([os.path.dirname(cairo.__path__[0])])
        if status != 0:
>           raise Exception("\n" + "\n".join([out, err]))
E           Exception: 
E           cairo/__init__.pyi: error: Duplicate module named 'cairo' (also at '/var/tmp/paludis/build/dev-python-pycairo-1.20.0/work/PYTHON_ABIS/3.9/pycairo-1.20.0/build/lib.linux-x86_64-3.9/cairo/__init__.pyi')
E           cairo/__init__.pyi: note: Are you missing an __init__.py? Alternatively, consider using --exclude to avoid checking one of them.
E           Found 1 error in 1 file (errors prevented further checking)

tests/test_typing.py:18: Exception
========================================================================= short test summary info ==========================================================================
FAILED tests/test_typing.py::test_mypy - Exception: 
================================================================= 1 failed, 275 passed, 1 skipped in 8.14s =================================================================
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -33,7 +33,7 @@ jobs:
       - name: Install dependencies
         shell: msys2 {0}
         run: |
-          python -m pip install pytest coverage hypothesis mypy
+          python -m pip install pytest coverage hypothesis
 
       - name: Run tests
         shell: msys2 {0}
@@ -68,6 +68,7 @@ jobs:
     - name: Run tests
       run: |
         poetry run python setup.py test
+        poetry run mypy .
 
   ubuntu:
     runs-on: ubuntu-latest
@@ -89,9 +90,6 @@ jobs:
         sudo apt-get install -y libcairo2-dev
         python -m pip install --upgrade setuptools
         python -m pip install --upgrade pytest flake8 sphinx sphinx_rtd_theme coverage hypothesis attrs wheel
-        if [[ "${{ matrix.python-version }}" != "pypy3" ]]; then
-          python -m pip install --upgrade mypy
-        fi
         if [[ "${{ matrix.python-version }}" != "pypy3" ]] && [[ "${{ matrix.python-version }}" != "3.8" ]] && [[ "${{ matrix.python-version }}" != "3.9.0-rc.1 - 3.9" ]]; then
           python -m pip install --upgrade pygame
         fi
@@ -148,7 +146,6 @@ jobs:
           python -m pip install --upgrade wheel
           python -m pip install --upgrade setuptools
           python -m pip install --upgrade pytest flake8 sphinx sphinx_rtd_theme coverage hypothesis attrs
-          python -m pip install --upgrade mypy || true
           python -m pip install --upgrade pygame
           python -m coverage run --branch setup.py test
           python -m coverage xml -i
@@ -180,7 +177,6 @@ jobs:
           python -m pip install --upgrade wheel
           python -m pip install --upgrade setuptools
           python -m pip install --upgrade pytest flake8 sphinx sphinx_rtd_theme coverage hypothesis attrs
-          python -m pip install --upgrade mypy || true
           python -m pip install --upgrade pygame
           python -m coverage run --branch setup.py test
           python -m coverage xml -i
@@ -220,7 +216,6 @@ jobs:
           source ../venv/bin/activate
           python -m pip install --upgrade setuptools
           python -m pip install --upgrade pytest flake8 sphinx sphinx_rtd_theme coverage hypothesis attrs wheel
-          python -m pip install --upgrade mypy || true
           # run tests
           python -m coverage run --branch setup.py test
           python -m coverage xml -i

--- a/tests/test_typing.py
+++ b/tests/test_typing.py
@@ -1,21 +1,8 @@
 import os
 import io
-import sys
 import types
 
 import cairo
-import pytest
-
-mypy = pytest.importorskip("mypy.api")
-pytestmark = pytest.mark.skipif(
-    sys.version_info[:2] < (3, 6), reason="Py3.6 only")
-pytestmark
-
-
-def test_mypy():
-    out, err, status = mypy.run([os.path.dirname(cairo.__path__[0])])
-    if status != 0:
-        raise Exception("\n" + "\n".join([out, err]))
 
 
 def test_typing():

